### DISCUSSION TITLE
Add a "Memorable" method to the password generator

### DISCUSSION THREAD

**Discussion by tinogithub** [keepassxreboot/keepassxc#7427] [Ideas]
Hello,
first of all, thank you for your efford to develop this program. I've used SafeInCloud for some years now however they don't support Linux and don't allow full functionality if you don't have an Google or Apple Store Account.
One thing however that I have used all the time is the password generator's method "Memorable" which is a combination of a passphrase and random chars. I have attached two screenshots.
![examples01](https://user-images.githubusercontent.com/23121904/154675956-e3c1a7e6-3bf7-47ba-89eb-90245da00fad.JPG)
![examples02](https://user-images.githubusercontent.com/23121904/154675973-61a83a4e-2a65-4e47-8095-50f3027c402e.JPG)

As I've found it useful to be able to remember some passwords that I need a lot, I used this method very often.
Because many security methods don't allow only letters, your passphrase generator doesn't fullfill these requirements. And the normal password generator only creates non-memorable passwords.

Has there been any discussion about this yet?

Thank you again.

**Comment by droidmonkey** [comment]
Use the passphrase tab. https://keepassxc.org/docs/KeePassXC_UserGuide.html#_generating_passphrases

**Reply by tinogithub** [reply]
Thank you for that quick reply.
However as I mentioned below the screenshots, the passphrase generator only produces letters which results in a password that is often not accepted by platforms.

**Reply by tinogithub** [reply]
Now I get what you mean. I have to enter the special characters manually as a word seperator.
Ok, this idea can be closed.

Thank you again.
